### ISSUE TITLE
Differences between CSIRO MOM5 and harmonised MOM5 outputs

### ISSUE THREAD

**Issue by peterdobro** [mom-ocean/MOM5#258]
We are seeing, differences in the ocean outputs between the existing CSIRO MOM5 code and the new harmonised MOM5 in ACCESS-CM2 tests. For example, the potential temperature differs regionally by several degrees in the first month.

Harmonised MOM5 outputs at /short/p66/pbd562/archive/av630/history/ocn can be compared with the existing CM2 tests with CSIRO MOM5 at /g/data1a/p66/dhb599/ACCESS-CM2/archive/av630/history/ocn/

The corresponding namelists can be found in /short/p66/pbd562/cylc-run/u-av630/work (new harmonised) and /short/p66/dhb599/cylc-run/u-av630/work (existing CSIRO).

I will be away until Friday next week, 16 Nov 2018, but please contact Dave or Arnold if you need read permissions on anything. Data are readable by p66.

Thanks, Peter

**Comment by russfiedler** [comment]
@peterdobro  You need to set `do_langmuir_cvmix=.true.` in the kpp namelist. It's using the Smythe version instead.

**Comment by peterdobro** [comment]
Thanks @russfiedler - I will give it a go.

**Comment by aidanheerdegen** [comment]
The verification plots etc for the code changes were done in the pull request:

https://github.com/mom-ocean/MOM5/pull/229

I note that the colour scale is not entered on zeros, so the large changes are all two patches off the coast of Greenland, which is where some of the greatest effects did show up in the plots from https://github.com/mom-ocean/MOM5/pull/229. 

Which is the original run and which is the one with the new code? I guess av630.v3 is harmonised and av630 is not? So the anomalies as plotted show a warming in those two patches off Greenland?

**Comment by peterdobro** [comment]
Yes that's right: av630.v3 is harmonised and av630 is not. And the namelists can be found in /short/p66/pbd562/cylc-run/u-av630.v3/work/... and /short/p66/dhb599/cylc-run/u-av630/work/... respectively.

**Comment by ofa001** [comment]
Looking at the Greenland anomaly region I would have thought the sea ice was playing a role amplifying the change, its January and at 5m near the surface but the SO has widespread anomalies well away from the  ice edge so it ahs to be something more.
